src/models/groups.model.ts

```diff
@@ -0,0 +1,49 @@
+import { DataTypes, type Sequelize } from 'sequelize'
+import { Model, Optional, ModelDefined } from 'sequelize'
+
+interface GroupAttributes {
+  id: number
+  name: string
+}
+// You can also set multiple attributes optional at once
+interface GroupCreationAttributes extends Omit<GroupAttributes, 'id'> {}
+
+export default class Group {
```

Comment:
I'd suggest using Sequelize' [recommended way of working with models](https://sequelize.org/docs/v6/other-topics/typescript/#usage) in Typescript. It's slightly different but much cleaner type-wise.